### ISSUE TITLE
question on the cross validation error logging in the training

### ISSUE THREAD

**Issue by funguy74** [Microsoft/CNTK#634]
For ndl file, two nodes are usually defined for logging during training:

cr = CrossEntropyWithSoftmax(labels, LSTMoutputW,tag=Criteria);
Err = ErrorPrediction(labels,LSTMoutputW,tag=Eval);

One is training cost function and the other is for accuracy (cluster error for my case). My question is that can errorPrediction use separate cross validation data for error prediction?

For example, for learning rate adaptation, cntk configuration can set up cvReader for separate cross validation data set and whenever criterion is not met based on the separate cv set, learning rate chages. 

I'm interested in display prediction error based on the cross validation set as well. It would be helpful to see current training is biased or overfit.

If errorPrediction cannot be used for that purpose, I want to know if there is any other way around.

**Comment by dongyu888** [comment]
Yes,  set  useEvalCriterionControlLR to true.

**Comment by funguy74** [comment]
Thanks for quick response. 

useEvalCriterionControlLR parameter when I read in WiKi is to choose between evaluation and training criterion. So. by its description, I'm a bit confusing.

Does it mean that it chooses between training data and cross validation data (set by cvReader) for learning rate adaptation and ErrorPrediction display?

**Comment by dongyu888** [comment]
No, it uses errorPrediction on the CV data to control the learning rate. I think it should also display errorPrediction on CV.

**Comment by funguy74** [comment]
Thanks for clarification. I'll test it by turning on this variable.
